tests/gis_tests/distapp/tests.py

```diff
@@ -645,6 +647,18 @@ def test_distance_order_by(self):
         ).values_list('name', flat=True).filter(name__in=('San Antonio', 'Pearland'))
         self.assertSequenceEqual(qs, ['San Antonio', 'Pearland'])
 
+    @skipUnless(postgis, "This is a PostGIS-specific test")
```

Comment:
I think we could skip this test, as it seems a bit fragile and not really needed given the other tests.